pages/linux/braa.md

```diff
@@ -0,0 +1,16 @@
+# braa
+
+> Ultra-fast mass SNMP scanner allowing multiple hosts simultaneously.
+> More information: <https://github.com/mteg/braa>.
+
+- Walk the SNMP tree of host with public string querying all OIDs under .1.3.6:
+
+`braa public@{ip}:.1.3.6.*`
```

Comment:
Did you want to use the placeholder syntax to highlight `ip`? Or does the command require the `{ }`?
Also, we use the placeholder syntax for all variable arguments, even if it's already a specific example.
```suggestion
`braa public@{{ip}}:{{.1.3.6.*}}`
```